### DISCUSSION TITLE
USB Issue

### DISCUSSION THREAD

**Discussion by floydp386** [MoleMan1024/audiowagon#106] [General]
2 quick questions:

1. How do you force stop the USB app to stop the warning message USB Stick Error - "This USB Stick is not supported" every time the car is switched on?

2. How do you "Clear default in System settings > Apps > Downloaded?
I can't find Apps in system settings

![20230124_193446](https://user-images.githubusercontent.com/116339566/214511933-be9a14a7-77d3-4aca-9490-a7942192e062.jpg)

**Comment by MoleMan1024** [comment]
[floydp386](https://github.com/floydp386) :

1)
[see FAQ](https://moleman1024.github.io/audiowagon/faq.html#why-is-there-a-popup-abbout-usb-device-is-not-supported-when-the-usb-device-works-fine).
To stop this app Android Settings > Apps & Notifications > Show all apps > System > USB status monitor > Force stop. However stopping this USB system app is only temporary, it will come back the next time Android is booted.

2) This setting does not exist, the info in the popup is wrong. Google or Volvo/Polestar need to change something to fix this.

**Reply by floydp386** [reply]
Thank you for the reply
